### PR TITLE
I implement tableLayout and tableRow part

### PR DESCRIPTION
**Pull request by tvroom88**

### CODE REVIEW

File: app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt [hunk 1]
```diff
@@ -1,31 +1,85 @@
...
+            savedInstanceState: Bundle?
     ): View? {
         homeViewModel =
-            ViewModelProviders.of(this).get(HomeViewModel::class.java)
+                ViewModelProviders.of(this).get(HomeViewModel::class.java)
+
+
+
         val root = inflater.inflate(R.layout.fragment_home, container, false)
-//        val textView: TextView = root.findViewById(R.id.text_home)
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt:25]: Can you remove unused code? (commented code)

File: app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt [hunk 2]
```diff
@@ -1,31 +1,85 @@
...
-        })
+        val tableLayout = root.findViewById<TableLayout>(R.id.TableLayout1)
+
+//        tableRow = root.findViewById<TableRow>(R.id.table_row1)
+        readData(myGogiyumAddress, tableLayout)
+
         return root
     }
+
+//    synchronous
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt:43]: This is not synchronous.

File: app/src/main/res/layout/custom_table_row.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            android:layout_width="50dp"
+            android:layout_height="50dp"
+            android:id="@+id/imageView1"/>
+    <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/textView4"
+            android:textSize="14sp"
+            android:textStyle="bold"/>
+</LinearLayout>
```
Comment (jaehoonLee) [inline app/src/main/res/layout/custom_table_row.xml]: Always add nextline.

File: app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt [hunk 4]
```diff
@@ -1,31 +1,85 @@
...
+            }
+        })
+    }
+
+    fun handleTableRow(jsonArray: JSONArray, tableLayout: TableLayout ){
+        for (i in 0 until jsonArray.length()) {
+            val jsonObject = jsonArray.getJSONObject(i)
+                val tableRows = TableRow(activity)
+                layoutInflater.inflate(R.layout.custom_table_row, tableRows, true)
+                tableRows.textView4.text = jsonObject.getString("name")
```
Comment (jaehoonLee) [inline app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt]: textView4 is not good naming. 
Can you change better name? 
i.e) foodImageName

File: app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt (hunk 1 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt:25]: I erased it

File: app/src/main/res/layout/custom_table_row.xml (hunk 3 above)
Comment (tvroom88) [reply app/src/main/res/layout/custom_table_row.xml]: I add nextline

File: app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt (hunk 4 above)
Comment (tvroom88) [reply app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt]: I changed it

**Comment by jaehoonLee** [comment]
Nice Job! LGTM (Looks good to me)

**Pull request merged**